### PR TITLE
add subinterface functionality

### PR DESCRIPTION
**Pull request by mmailand**
add subinterface functionality

### CODE REVIEW

**Comment by jerearista** [comment]
Jenkins, test this please

**Comment by jerearista** [comment]
Jenkins, retest this please

**Comment by jerearista** [comment]
Jenkins, retest this please

**Comment by jerearista** [comment]
Jenkins, retest this please (testing some job updates)

File: spec/system/rbeapi/api/interfaces_portchannel_spec.rb [hunk 1]
```diff
@@ -82,6 +106,19 @@
     end
   end
 
+=begin THIS TEST BREAKS OTHER TESTS
```
Comment (jerearista) [inline spec/system/rbeapi/api/interfaces_portchannel_spec.rb]: How does this break other tests?  Let's look at options to get it working.

File: lib/rbeapi/api/interfaces.rb [hunk 2]
```diff
@@ -229,6 +254,7 @@ def parse_shutdown(config)
       #
       # @return [Hash<Symbol, Object>] Returns the resource hash attribute.
       def parse_load_interval(config)
+        #mdata = /^\s{3}load-interval\s(\.+)$/.match(config)
```
Comment (mmailand) [inline lib/rbeapi/api/interfaces.rb]: done

File: spec/spec_helper.rb [hunk 3]
```diff
@@ -27,4 +27,9 @@
   config.mock_with :rspec do |rspec_config|
     rspec_config.syntax = :expect
   end
+
+  config.after do
+    # TODO Run copy star ru at the end of the test to cleanup the mess
```
Comment (mmailand) [inline spec/spec_helper.rb]: removed that, going to put that in to a diffrent PR

File: spec/system/rbeapi/api/interfaces_portchannel_spec.rb (hunk 1 above)
Comment (mmailand) [inline spec/system/rbeapi/api/interfaces_portchannel_spec.rb]: Fix the test.
the Problem was, that the return value of the parse_lacp_timeout method for portchannel was not defined, therefore the tests didn't work. 
Fixed that.

**Pull request merged**